src/scripts/content/microsoft-to-do.js

```diff
@@ -16,12 +16,12 @@ togglbutton.render('.taskItem-body:not(.toggl)', { observe: true }, function (
       : '';
 
   const link = togglbutton.createTimerLink({
-    className: 'microsoft-todo',
+    className: 'ms-todo-list',
```

Comment:
⚠️ let's not change this yet, it'll break analytics data consistency *(yeah..)*. Save that disruption for #1292 😁

Reply:
Restored :+1: